egs/fame/s5/local/score_mbr.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
this file can be removed as far as I can tell, as the score_kaldi.sh can do MBR decoding as well.